core/tests/protractor_utils/TopicsAndSkillsDashboardPage.js

```diff
@@ -120,10 +133,17 @@ var TopicsAndSkillsDashboardPage = function() {
   };
 
   this.navigateToTopicWithIndex = async function(index) {
-    var elem = await topicsListItems.get(index);
+    await waitFor.visibilityOf(topicsTable,
+      'Topic table taking too long to appear.');
+    var editTopicButton = element(
+      by.css('.protractor-test-edit-topic-button'));
+    var topicEditOptionBox = await topicEditOptions.get(index);
```

Comment:
Don't need to await `.get()` because `.get()` returns an ElementFinder, not a Promise

Reply:
Done.